index.php

```diff
@@ -41,6 +43,12 @@
 
 admin_externalpage_setup('reportallbackups', '', array('tab' => $currenttab), '', array('pagelayout' => 'report'));
 
+global $CFG;
```

Comment:
this should not be using global $CFG - this setting is named incorrectly, plugins must store their settings in the config_plugins table and use get_config() to access. This string should also be in the lang pack and ideally throwing a moodle_exception rather than just echoing text and dying.